packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -59,10 +59,6 @@
 							:output="selectedOutputId"
 							@update:selection="(e) => emit('update:selection', e)"
 						/>
-						<section v-if="!isEmpty(menuItems)" class="mx-2">
-							<Button icon="pi pi-ellipsis-v" rounded text @click.stop="toggleEllipsisMenu" />
-							<Menu ref="ellipsisMenu" :model="menuItems" popup />
```

Comment:
You can remove the prop menuItems now that is not used anymore.

Reply:
<img width="410" alt="image" src="https://github.com/user-attachments/assets/b536729e-2d2f-4614-8869-7c093c417922">